clients/src/main/java/org/apache/kafka/common/protocol/Readable.java

```diff
@@ -38,6 +38,7 @@ public interface Readable {
     int readVarint();
     long readVarlong();
     int remaining();
+    Readable slice();
```

Comment:
Can we add Java Doc comment to this new method. `Readable` is an important type and it should be fully documented but I understand that is may be out of scope of this PR.

Reply:
Done